scripts/backend_test_shards.json

```diff
@@ -351,6 +351,7 @@
         "core.domain.question_domain_test",
         "core.platform.bulk_email.mailchimp_bulk_email_services_test",
         "core.platform.bulk_email.dev_mode_bulk_email_services_test",
-        "core.jobs.io.job_io_test"
+        "core.jobs.io.job_io_test",
+        "core.jobs.batch_jobs.exp_version_history_computation_job_test"
```

Comment:
Shard 4 is taking 34 mins, which is longer than our expected 30 mins. Please add this test to a shard with a shorter runtime to load-balance across the shards

Reply:
I have changed the shard to "3" now. Let's see if this resolves the issue.

Reply:
Shifting the test to the second shard runs under 30 mins.